client/src/mongoose.js

```diff
@@ -0,0 +1,70 @@
+// const mongoose = require('mongoose')
```

Comment:
You can remove these files that aren't used to cut down on clutter in your directory